.github/CODEOWNERS

```diff
@@ -49,6 +49,7 @@
 /libs/barista-components/selection-area/** @ffriedl89 @lukasholzer
 /libs/barista-components/show-more/** @rowa-audil
 /libs/barista-components/stepper/** @ffriedl89 @thomaspink
+/libs/barista-components/sunburst-chart/** @subarroca @pedromosquera
```

Comment:
@ffriedl89 @tomheller should the second required review be one from the barista core team?

Reply:
We added @PedroMosquera just because we needed a second one. If @bardosmisi is open to be the second code owner we could remove Pedro

Reply:
you can add 3 as well it has only to be at least one of the core team members.